src/lib/sitemap/tutorials-content-fields.ts

```diff
@@ -0,0 +1,50 @@
+import { getAllCollections } from 'lib/learn-client/api/collection'
+import { SectionOption } from 'lib/learn-client/types'
+import { activeProductSlugs } from 'lib/products'
+import { generateTutorialMap } from 'pages/api/tutorials-map'
+import { ProductSlug } from 'types/products'
+import { getCollectionSlug } from 'views/collection-view/helpers'
+import { makeSitemapField } from './helpers'
+import { Collection as ClientCollection } from 'lib/learn-client/types'
+
+function getTutorialLandingPaths(): string[] {
+	const activeSlugs = activeProductSlugs.map(
+		(productSlug: ProductSlug) => `/${productSlug}/tutorials`
+	)
+	const sectionOptionsWithLandingPage = Object.values(SectionOption).filter(
+		(option: SectionOption) => option === SectionOption.onboarding
```

Comment:
```suggestion
		(option: SectionOption) => option !== SectionOption.onboarding
```
Oops logical error on my part!